docker/settings.py

```diff
@@ -19,10 +19,11 @@
                        'u_acadev_provision_support'],
     }
 
-if os.getenv("ENV") == "prod":
+if not os.getenv("ENV") == "localdev":
     INSTALLED_APPS += ['rc_django',]
-    RESTCLIENTS_DAO_CACHE_CLASS='endorsement.cache.ProvisionCache'
-    APP_SERVER_BASE = 'https://provision.uw.edu'
+    RESTCLIENTS_DAO_CACHE_CLASS = 'endorsement.cache.ProvisionCache'
+    if os.getenv("ENV") == "prod":
+        APP_SERVER_BASE = 'https://provision.uw.edu'
```

Comment:
What about provision-test?

Reply:
could probably be clearer, but it defaults in models/core.py:277 since we didn't want a null hosted url